src/components/sbb-link/sbb-link.scss

```diff
@@ -39,26 +39,26 @@
   }
 }
 
-.sbb-link--block-negative,
-.sbb-link--inline-negative {
-  --sbb-focus-outline-color: var(--sbb-focus-outline-color-dark);
+button.sbb-link {
+  @include buttonReset;
 }
 
-.sbb-link--block-negative {
+.sbb-link--negative {
+  --sbb-focus-outline-color: var(--sbb-focus-outline-color-dark);
   --text-color-normal: var(--sbb-color-white-default);
   --text-color-hover-focus: var(--sbb-color-cloud-default);
   --text-color-active: var(--sbb-color-silver-default);
 }
 
 .sbb-link--inline,
-.sbb-link--inline-negative {
+.sbb-link--inline.sbb-link--negative {
```

Comment:
Is line 54 really needed? Without testing I think .sbb-link--inline is sufficient here.

Reply:
True words, thx! :)